MerchantSDK/GiniMerchantSDK/Sources/GiniMerchantSDK/Core/PaymentComponentsController.swift

```diff
@@ -690,6 +675,10 @@ extension PaymentComponentsController: InstallAppBottomViewProtocol {
 }
 
 extension PaymentComponentsController: PaymentReviewProtocol {
+    public func presentShareInvoiceBottomSheet(paymentRequestId: String, paymentInfo: GiniInternalPaymentSDK.PaymentInfo) {
+        //
```

Comment:
[clarification] Nothing to be added here. Right?

Reply:
Yes. Is a addopted protocol for GiniMerchant that is going to be fixed later. GiniMerchant isn't subject of release. We are going to refactor that SDK soon